start_group.html

```diff
@@ -0,0 +1,888 @@
+                                        <h6 class="cmp-title__text">MAKE A DIFFERENCE</h6>
+                                    </div>
+                                </div>
+                                <div class="title aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                    <div id="title-c6c8e598f8" class="cmp-title">
+                                        <h3 class="cmp-title__text">Why Start a Group?</h3>
+                                    </div>
+                                </div>
+                                <div class="text aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                    <div id="text-924f8b967f" class="cmp-text">
+                                        <p>Starting a LeaderImpact group is an amazing experience. If you choose to start a group, you’ll have the opportunity to see lives change, leaders making a difference because of your influence, and have new avenues to reach other leaders in your community personally, professionally and spiritually.</p>
+                                    </div>
+                                </div>
+                                <div class="button aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                    <a id="button-254edd7bf9" class="cmp-button" href="">
+                                        <span class="cmp-button__text">Apply Now</span>
+                                    </a>
+                                </div>
+
+                            </div>
+                        </div>
+                        <div
+                            class="container responsivegrid video-wrap aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div data-cmp-data-layer="{&quot;embed-2aa125dec4&quot;:{&quot;@type&quot;:&quot;core-components-examples/components/embed&quot;}}"
+                                id="embed-2aa125dec4" class="cmp-embed">
+                                <div class="cmp-embed__youtube">
+                                    <iframe class="cmp-embed__youtube-iframe" title="youtube video"
+                                        data-cmp-data-layer="{&quot;youtube-2aa125dec4&quot;:{&quot;@type&quot;:&quot;core/wcm/components/embed/v1/embed/embeddable/youtube&quot;,&quot;embeddableProperties&quot;:{&quot;youtubeVideoId&quot;:&quot;f7hbWvHKns0&quot;}}}"
+                                        src="https://www.youtube.com/embed/7jMlFXouPk8" frameborder="0"
+                                        allowfullscreen="" allow="autoplay; fullscreen"
+                                        aria-label="YouTube Video"></iframe>
+                                </div>
+                            </div>
+                        </div>
+
+                    </div>
+                </div>
+            </div>
+        </div>
+      </div>
+      <!-- START GROUP SLIDER -->
```

Comment:
@Prystal254 Was there instruction outside of Figma comments saying to make these tiles a slider?